build/ios/scripts/build_arch.sh

```diff
@@ -3,48 +3,41 @@ set -e
 
 if [ $# -ne 2 ]; then
     echo "$0 platform arch" 1>&2
-    echo "Example: $0 iPhoneSimulator i386"
+    echo "Example: $0 iphonesimulator i386"
     exit 1
 fi
 
 PLATFORM=$1
 ARCH=$2
 
-if [ "$PLATFORM" = "iPhoneOS" ]; then
-    EXTRA_CONFIG="--host=arm-apple-darwin14 --target=arm-apple-darwin14"
+MINIOSVERSION="7.1"
+if [ "$PLATFORM" = "iphoneos" ]; then
+    EXTRA_CONFIG="--host=arm-apple-darwin14 --target=arm-apple-darwin14 --enable-static --disable-shared"
+    MINVERSION="-miphoneos-version-min=$MINIOSVERSION"
 elif [ "$ARCH" = "i386" ]; then
-    EXTRA_CONFIG="--host=i386-apple-darwin14 --target=i386-apple-darwin14"
+    EXTRA_CONFIG="--host=i386-apple-darwin14 --target=i386-apple-darwin14 --enable-static --disable-shared"
+    MINVERSION="-mios-simulator-version-min=$MINIOSVERSION"
 elif [ "$ARCH" = "x86_64" ]; then
-    EXTRA_CONFIG="--host=x86_64-apple-darwin14 --target=x86_64-apple-darwin14"
+    EXTRA_CONFIG="--host=x86_64-apple-darwin14 --target=x86_64-apple-darwin14 --enable-static --disable-shared"
+    MINVERSION="-mios-simulator-version-min=$MINIOSVERSION"
 else
     echo "$0: unsupported configuration" 1>&2
     exit 1
 fi
 
 ROOTDIR=$(cd $(dirname $0) && pwd -P)
 SOURCEDIR="$ROOTDIR/../../../"
-BUILDDIR="$ROOTDIR/../"
-
-# XXX We assume that the user has the iphoneos sdk installed
-AVAIL_SDKS=`xcodebuild -showsdks | grep "iphoneos"`
-FIRST_SDK=`echo "$AVAIL_SDKS" | head -n1`
-SDKVERSION=`echo "$FIRST_SDK" | cut -d\  -f2`
-echo "Using this SDK version: $SDKVERSION"
-
-DEVELOPER=$(xcode-select -print-path)
-MINIOSVERSION="7.1"
-
-export PATH="${DEVELOPER}/Toolchains/XcodeDefault.xctoolchain/usr/bin/:${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer/usr/bin/:${DEVELOPER}/Toolchains/XcodeDefault.xctoolchain/usr/bin:${DEVELOPER}/usr/bin:${PATH}"
+DESTDIR="$ROOTDIR/../build/${PLATFORM}/${ARCH}/"
 
-export CC="/usr/bin/gcc -arch ${ARCH} -miphoneos-version-min=${MINIOSVERSION}"
-export CXX="/usr/bin/g++ -arch ${ARCH} -miphoneos-version-min=${MINIOSVERSION}"
-export CPPFLAGS="-isysroot ${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer/SDKs/${PLATFORM}${SDKVERSION}.sdk"
-export CFLAGS="-isysroot ${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer/SDKs/${PLATFORM}${SDKVERSION}.sdk"
-export CXXFLAGS="-isysroot ${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer/SDKs/${PLATFORM}${SDKVERSION}.sdk"
-export LDFLAGS="-isysroot ${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer/SDKs/${PLATFORM}${SDKVERSION}.sdk"
+export CC="$(xcrun -find -sdk ${PLATFORM} cc)"
+export CXX="$(xcrun -find -sdk ${PLATFORM} g++)"
```

Comment:
Okay, I see your point towards consistency with GNU's standard. My concern was that `gcc -arch ${ARCH}` could be considered "the compiler". But, given that your version builds as well, I am going to accept this hunk, since it's less surprising than before. Thanks.